lib/new_relic/telemetry_sdk/clients/client.rb

```diff
@@ -19,6 +23,10 @@ def initialize host:,
         @gzip_request = use_gzip
         @payload_type = payload_type
         add_content_encoding_header @headers if @gzip_request
+        @connection_attempts = 0
+        @max_retries= 8 # based on config
+        @backoff_factor = 5 # based on config
+        @backoff_max = 80 # based on config
```

Comment:
Oh that is surprising because I mostly read the [backoff section](https://github.com/newrelic/newrelic-telemetry-sdk-specs/blob/master/communication.md#graceful-degradation) and the way it talks about backoff_max it seemed configurable. There doesn't seem to be any given value or default. 
I agree that maybe the specs could be a little more clear here.

Reply:
Yeah, maybe it should make it into the aforementioned config section.

Reply:
Alright, I've submitted a spec PR to get this added to the config section. Thank you for bringing that to my attention!

Reply:
Thank you for submitting the spec PR!